### PR TITLE
Fix openapi keys

### PR DESCRIPTION
**Pull request by tuwilof**

### CODE REVIEW

File: lib/tomograph/openapi/openapi2.rb [hunk 1]
```diff
@@ -28,15 +28,15 @@ def responses(resp, defi)
         resp.inject([]) do |result, reponse|
           if reponse[1]['schema']
             result.push(
-              status: reponse[0],
-              body: schema(reponse[1]['schema'], defi),
-              'content-type': ''
+              'status' => reponse[0],
+              'body' => schema(reponse[1]['schema'], defi),
+              'content-type' => 'application/json'
```
Comment (ngoral) [inline lib/tomograph/openapi/openapi2.rb:33]: Why should there be an `application/json`?
In case I have something like
```yml
          content:
            text/html:
              schema:
                type: string
```
it won't be parsed correctly.
I just faced with it while using `fitting` gem. And it took some time to dig to this place.

There also can be a `$ref` instead of a schema itself.
https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#responses-object

If you want, I can try to make a PR, just need to know, why here and in the next branch there are `application/json`...

File: lib/tomograph/openapi/openapi3.rb [hunk 2]
```diff
@@ -28,15 +28,15 @@ def responses(resp, defi)
           if response[1]['content'].nil?
             # TODO: 403Forbidden
             result.push(
-              status: response[0],
-              body: {},
-              'content-type': ''
+              'status' => response[0],
+              'body' => {},
+              'content-type' => 'application/json'
```
Comment (ngoral) [inline lib/tomograph/openapi/openapi3.rb:33]: Why should there be an `application/json`?
In case I have something like
```yml
          content:
            text/html:
              schema:
                type: string
```
it won't be parsed correctly.
I just faced with it while using `fitting` gem. And it took some time to dig to this place.

There also can be a `$ref` instead of a schema itself.
https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#responses-object

If you want, I can try to make a PR, just need to know, why here and in the next branch there are `application/json`...

Comment (tuwilof) [reply lib/tomograph/openapi/openapi3.rb:33]: Thank you for your questions and interest in libraries.

Before content-type was not used much in the fitting, it was used as a stub. Yes, it is desirable to correct your remark.

About $ref, fitting works according to the json schema, but openapi has a link outside the json schema. Further in the tomograph code, links outside the json-schema are replaced with links inside the json-schema or their contents are simply transferred, if this does not work for your case, then improvements are needed, details are needed here, an example of documentation. 

It will be cool if you can fix it. You can add an example of a test or input data for a negative case, so it will be easier for me to help if necessary.

You can use the forks from the master branch and the github functionality to design a pull request.

Comment (ngoral) [reply lib/tomograph/openapi/openapi3.rb:33]: Thank you for quick response!
I'll fork and try to create a PR as usual, and if have some doubts, check with you, thanks!

To be honest, I didn't quite underastand your explanation about json-schema and links.

I was talking about a case when I have something like this in my docs:
```yml
...
    responses:
      '404':
        $ref: '#/components/responses/NotFound'
```

It is valid according to a specification: the value of a `HTTP Status Code` property can be either `Response Object` or a `Reference Object`. But as I can see from the code, it can not process this case correctly. (I may be wrong though.)

Comment (ngoral) [reply lib/tomograph/openapi/openapi3.rb:33]: Ah, I got it about json-schema and openapi working outside of it. I'll get a closer look, thanks
